staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2163,6 +2179,24 @@ type RemoveVolumeOptions struct {
 	DryRun []string `json:"dryRun,omitempty"`
 }
 
+// AddInterfaceOptions is provided when dynamically hot plugging a network interface
+type AddInterfaceOptions struct {
+	// NetworkName indicates the name of the network to which the interface will be connected
+	NetworkName string `json:"networkName"`
+
+	// InterfaceName indicates the name of the interface being plugged into the guest
+	InterfaceName string `json:"interfaceName"`
```

Comment:
I would use the same description as the original [Interface](https://github.com/kubevirt/kubevirt/blob/main/staging/src/kubevirt.io/api/core/v1/schema.go#L1150) has.
```
"InterfaceName indicates the logical name of the interface being plugged."
```
I would drop the `into the guest` part since it may create a confusion that this name will be the name of the interface in the guest.